### PR TITLE
check that if there is any mistake in this code

### PR DESCRIPTION
**Pull request by mragab2000**

### CODE REVIEW

File: oop/oop.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
```
Comment (mohamedragabdahab) [inline oop/oop.php:1]: Please add strict type

File: oop/oop.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+class Cars
+{
+    public string $brand;
+    public string $model;
+    public int $releaseDate;
```
Comment (mohamedragabdahab) [inline oop/oop.php:6]: Int is wrong datatype, there is DateTime one, google it

File: oop/oop.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    public function autoDriving(bool $ad): bool
```
Comment (mohamedragabdahab) [inline oop/oop.php:9]: What $ad variable means?

File: oop/oop.php [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    {
+        if ($ad == true) {
+
+        }
+    }
+    public function autoParking(bool $ap): bool
+    {
+        if ($ap == true) {
```
Comment (mohamedragabdahab) [inline oop/oop.php:17]: What $ap means?

File: oop/oop.php (hunk 3 above)
Comment (mragab2000) [reply oop/oop.php:9]: it stands for auto driving

File: oop/oop.php (hunk 4 above)
Comment (mragab2000) [reply oop/oop.php:17]: this for auto parking.

File: oop/oop.php (hunk 1 above)
Comment (mragab2000) [reply oop/oop.php:1]: fixed

**Pull request merged**